_data/variables.yml

```diff
@@ -850,132 +850,148 @@
-      description: Sets `'grid-gap-md'` output
+      description: Sets `'grid-gap-md'` output.
       var: $theme-column-gap-md
       default: 2
       kind: units
     - name: Large grid gap
-      description: Sets `'grid-gap-lg'` output
+      description: Sets `'grid-gap-lg'` output.
       var: $theme-column-gap-md
       default: 3
       kind: units
     - name: Mobile grid gap
-      description: Sets value of `'grid-gap'` at `mobile` width
+      description: Sets value of `'grid-gap'` at `mobile` width.
       var: $theme-column-gap-mobile
       default: 2
       kind: units
     - name: Desktop grid gap
-      description: Sets value of `'grid-gap'` at `desktop` width
+      description: Sets value of `'grid-gap'` at `desktop` width.
       var: $theme-column-gap-desktop
       default: 4
       kind: units
 
     - name: Grid container max width
       section: true
-      description: Sets the `max-width` value of the `.grid-container` utility
+      description: Sets the `max-width` value of the `.grid-container` utility.
       var: $theme-grid-container-max-width
       default: "'desktop'"
       kind: units
 
     - name: Site max width
       section: true
-      description: Used to set a consistent `max-width` across components
+      description: Used to set a consistent `max-width` across components.
       var: $theme-site-max-width
       default: "'desktop'"
       kind: units
     - name: Site outer margins breakpoint
-      description: At what breakpoint should the site's outer margin change
+      description: At what breakpoint should the site's outer margin change.
       var: $theme-site-margins-breakpoint
       default: "'desktop'"
       kind: units
-    - name: Site mobile outer margins
-      description: Site outer margin at mobile width
-      var: $theme-site-margins-mobile-width
-      default: 2
-      kind: units
     - name: Site outer margins
-      description: Site outer margin at non-mobile width (breakpoint defined by `$theme-site-margins-breakpoint`)
+      description: Site outer margin at non-mobile width (breakpoint defined by `$theme-site-margins-breakpoint`).
       var: $theme-site-margins-width
       default: 4
       kind: units
+    - name: Site mobile outer margins
+      description: Site outer margin at mobile width.
+      var: $theme-site-margins-mobile-width
+      default: 2
+      kind: units
 
 - section:
   name: Typography settings
-  description: Set values related to typefaces, type style, font size, and weights
+  description: Set values related to typefaces, type style, font size, and weights.
   contents:
     - name: Respect user font size
       description: |
-        When `true` the root font size will be `100%` and media queries use  ems. When `false`, the root font size will be the px value specified in `$theme-root-font-size` and media queries will use px values.
+        When `true` the root font size will be `100%` and media queries use  ems.
```

Comment:
- `font size will be` → `font size is`
- remove extra space between `use` and `ems`